validation/mounts.go

```diff
@@ -1,9 +1,26 @@
 package main
 
 import (
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/opencontainers/runtime-tools/validation/util"
 )
 
 func main() {
-	util.Skip("TODO: mounts generation options have not been implemented", "")
+	g := util.GetDefaultGenerator()
+	mount := rspec.Mount{
+		Destination: "/tmp",
+		Type:        "tmpfs",
+		Source:      "tmpfs",
+		Options: []string{
+			"nosuid",
+			"strictatime",
+			"mode=755",
+			"size=65536k",
```

Comment:
I'm not sure if mounting the tmpfs actually allocates the memory or not, but since this is just a test and we aren't writing anything to `/tmp` I'd rather use a smaller value (1k?).  On the other hand, 65 MB isn't *that* big.

Reply:
Fine, updated.